SCons/Tool/gdc.py

```diff
@@ -82,13 +84,17 @@ def generate(env) -> None:
 
     env['DINCPREFIX'] = '-I'
     env['DINCSUFFIX'] = ''
-    env['DVERPREFIX'] = '-version='
+    env['DVERPREFIX'] = '-fversion='
     env['DVERSUFFIX'] = ''
-    env['DDEBUGPREFIX'] = '-debug='
+    env['DDEBUGPREFIX'] = '-fdebug='
```

Comment:
was the previous value always wrong? Or did this change at some version of gdc?

Reply:
Always wrong.

Reply:
There is a gdmd wrapper for gdc that has the same arguments as dmd, but the arguments are different here.